puppet-client.html

```diff
@@ -162,16 +199,46 @@
                 value: '/'
             },
 
+            /**
+             * @private
```

Comment:
To reduce the overhead of publishing it to polymer notification protocol, as we do not expose and notify about each change. We already emit puppet events.